services/iota_wrappers.go

```diff
@@ -299,48 +309,51 @@ func GetProcessingFrequency() float64 {
 }
 
 // Finds Transactions with a list of addresses. Result in a map from Address to a list of Transactions
-func findTransactions(addresses []giota.Address) (map[giota.Address][]giota.Transaction, error) {
+func findTransactions(addresses []trinary.Hash) (map[trinary.Hash][]transaction.Transaction, error) {
 
-	addrToTransactionMap := make(map[giota.Address][]giota.Transaction)
+	addrToTransactionMap := make(map[trinary.Hash][]transaction.Transaction)
 
 	numOfBatchRequest := int(math.Ceil(float64(len(addresses)) / float64(MaxNumberOfAddressPerFindTransactionRequest)))
 
 	remainder := len(addresses)
 	for i := 0; i < numOfBatchRequest; i++ {
 		lower := i * MaxNumberOfAddressPerFindTransactionRequest
 		upper := i*MaxNumberOfAddressPerFindTransactionRequest + int(math.Min(float64(remainder), MaxNumberOfAddressPerFindTransactionRequest))
-		req := giota.FindTransactionsRequest{
+		req := giota.FindTransactionsQuery{
 			Addresses: addresses[lower:upper],
```

Comment:
It's somewhat surprising that `[]trinary.Hash` is an acceptable type to pass in to this.

Reply:
I think Address is backed by Hash(and by Trytes), so it makes sense to remove the Address type and just use Hash

Reply:
Okay, we don't have to worry about it in this PR.  But in a PR in the future, we should replace most uses of `trinary.Hash` with `trinary.Trytes`.  All hashes are tryte strings, but not all tryte strings are hashes, if that makes sense.  Everything will still work either way.